Alicerce/Sources/Logging/Providers/FileProvider.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  FileProvider.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 03/04/17.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+final class FileProvider {
+    
+    internal var minLevel: Log.Level = .error
+    internal var formatter: LogItemFormatter = LogItemStringFormatter()
+    
+    let fileURL: URL
+    let fileManager = FileManager.default
+    
+    //MARK:- Lifecycle
+    
+    init(fileURL: URL) {
+        self.fileURL = fileURL
+    }
+    
+    //MARK:- Public Methods 
+    
+    public func clear() {
```

Comment:
Shouldn't this call `throw`? I personally don't like "hiding" errors in log messages on a framework.